packages/ckeditor5-image/tests/image/imageblockediting.js

```diff
@@ -651,7 +572,7 @@ describe( 'ImageBlockEditing', () => {
 					'<imageBlock ' +
 						'src="/assets/sample.png" ' +
 						'alt="alt text" ' +
-						'srcset=\'{ "data": "small.png 148w, big.png 1024w", "width": "1024" }\'>' +
+						'srcset=\'{"data":"small.png 148w, big.png 1024w"}\'>' +
```

Comment:
Why is it not: `'srcset="small.png 148w, big.png 1024w">'`?

Reply:
Updated in b389967.